src/components/VueBootstrapAutocomplete.vue

```diff
@@ -21,7 +21,7 @@
         :id="`typeahead-input-${id}`"
         type="text"
         role="searchbox"
-        :class="`form-control ${inputClass}`"
+        :class="[ 'form-control', inputClass, state !== null && { 'is-invalid': state === false, 'is-valid': state === true } ]"
```

Comment:
```suggestion
        :class="[ 'form-control', inputClass, state !== null && { 'is-invalid': !state, 'is-valid': state } ]"
```

Can we simplify this line to this instead of explicit boolean checks or would that break it?